src/components/page-header.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+import { ReactNode } from "react";
+
+interface PageHeaderProps {
+  titleText: string;
+  subtitleText?: string;
+  subtitleElement?: ReactNode;
+}
+
+export default function PageHeader({
+  titleText,
+  subtitleText,
+  subtitleElement
+}: PageHeaderProps) {
+  return (
+    <>
+      <div
+        className="relative flex flex-col items-center justify-center text-center"
+        style={{ minHeight: "160px" }}
```

Comment:
this could be a tailwind class i.e. `min-h-40`

Reply:
I did try that, but for some reason it seems to be ignored. If I get rid of the style and add ``min-h-40`` to the classes, then when I go to the Core Services page (``/core-services``) the min height seems to be ignored since the header is only 84px tall. Do you get the same behaviour?

Reply:
Idk why it doesn't work. On the other hand, if I copy paste the same styles into the tailwind playground (https://play.tailwindcss.com/) it works as expected.

Reply:
so its actually because minimum height only has 3 classes in tailwind See [here](https://v2.tailwindcss.com/docs/min-height)

If you're just trying to set the height of the element to 160px, you would just use the height classes i.e. `h-40`

Reply:
We're using tailwind v3.4.1, not v2 though right? [These docs](https://tailwindcss.com/docs/min-height) do have ``min-h-40``.

Reply:
I don't think just setting the height will be good because on mobile the text in the services page overflows the header, so it needs to become taller.

Reply:
Ah fair enough then.